docs/pages/prometheus.md

```diff
@@ -4,4 +4,93 @@ title: Monitoring with Prometheus
+
+As a default, we suggest the following alerts:
+
+```
+groups:
+- name: runscanner.rules
+  rules:
+  - alert: StuckRuns
+    expr: sum_over_time(miso_runscanner_new_runs_scanned[1h]) - sum by(instance, environment,
+      job) (rate(miso_runscanner_waiting_runs[1h])) < 3 and time() - process_start_time_seconds  > 8 * 3600
+    annotations:
+      description: The runs being processed by {{$labels.instance}} are not being
+        processed and cleared.
+      summary: Runscanner {{$labels.instance}} seems to be stuck
+  - alert: BadRuns
+    expr: miso_runscanner_directories_attempted - miso_runscanner_directories_accepted
+      > 0
+    annotations:
+      description: The run scanner {{$labels.instance}} has found candidate sequencer
+        output directories that it does not have permission to read.
+      summary: Unreadable directories run directories on {{$labels.instance}}
+  - alert: AutoInhibit
+    expr: time() - process_start_time_seconds{job="runscanner"}  < 15 * 60 or sum(miso_runscanner_waiting_runs) by (environment, instance, job) - miso_runscanner_bad_runs > 5
+    labels:
+      scope: runscanner
+    annotations:
+      description: Run Scanner was restarted recently and probably needs time to finish scanning old runs
+      summary: Runscanner cache is cold
+```
+
+`StuckRuns` will fire when Run Scanner no longer seems to be making progress
+extracting data from run directories. This usually occurs if I/O latency has
+increased or CPU starvation.
+
+`BadRuns` will fire when there are unreadable due to permission errors. This
+requires human intervention to correct the directory permissions. Run Scanner
+will reattempt the affected runs.
+
+The `AutoInhibit` alert will fire after Run Scanner restarts until the cache is
+warm. Since it can take a long period of time for Run Scanner to start up and
+scan all available data, it maybe useful to stop applications from attempting
+access until the run cache is warm. To do this, have the application check for
+an `AutoInhibit` alert firing and wait until later. This alert should not be
+sent to humans. There is no action to be taken when firing.
+
+
+If using [MISO](https://github.com/miso-lims/miso-lims) to collect runs from
+Run Scanner, we suggest the following alert:
+
+```
+groups:
+- name: miso.rules
+  - alert: DroppedRuns
+    expr: miso_runscanner_client_bad_runs > 0
+    annotations:
+      description: The runs being received by MISO {{$labels.instance}} are not being
+        saved to the database.
+      summary: Runs are failing to save on {{$labels.instance}}
+```
+
+`DroppedRuns` will fire when runs are failing to save in MISO. This can happen
+for several reasons:
+
+* the run is for an instrument that is not registered in MISO. Add the
+  instrument to MISO or remove that path from Run Scanner's configuration.
+* there is a version mismatch between Run Scanner and MISO. Upgrade the lagging
+  software package.
```

Comment:
Could add a note to always check the release notes when upgrading MISO, as Run Scanner version changes should be noted there